### DISCUSSION TITLE
put  toc drawer icon on the top right of header on mobile

### DISCUSSION THREAD

**Discussion by jcyrss** [squidfunk/mkdocs-material#6886] [Q&A]
On mobile, now toc drawer icon is located on the right side of current nav item, we need to click 2 times to make toc appear. And it's even not easy to be found when nav item lists is long.

![image](https://github.com/squidfunk/mkdocs-material/assets/10496014/800051b8-dd76-4cf8-bef5-7d646588fb43)

How could I move it onto the top right of header bar?

**Comment by kamilkrzyskow** [comment]
Hello @jcyrss,
best I can do is to put the icon at the top of the navigation drawer with extra CSS:
https://squidfunk.github.io/mkdocs-material/customization/#additional-css
```css
.md-nav__link--active .md-nav__icon {
  position: absolute;
  top: 2rem;
  right: 2rem;
}
```
No matter the positioning setting, it's always constrained by the navigation drawer, so the only other solution I can think of would be to use JavaScript to add a custom TOC button in the position you want and invoke with it the logic of the left button, but I can't say if it's the best solution there is.
https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript

**Reply by jcyrss** [reply]
Thank you @kamilkrzyskow. At least by your suggestion of CSS way, the TOC drawer icon is easier to see now. I will take this for time being. :)

**Reply by squidfunk** [reply]
We will re-evaluate mobile navigation, including the table of contents, once we work on the visual appearance again, which is planned for later this year.

**Reply by jcyrss** [reply]
That's great.
